src/view/Data/Migrations/IdentityContextModelSnapshot.cs

```diff
@@ -178,24 +178,6 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .HasFilter("[NormalizedUserName] IS NOT NULL");
 
                     b.ToTable("AspNetUsers");
-
-                    b.HasData(
-                        new
-                        {
-                            Id = "7afb4539-8ab2-44d1-a881-2dbc8fdd7ab8",
-                            AccessFailedCount = 0,
-                            ConcurrencyStamp = "3d406bb9-460b-44ad-8bf0-513908f095f6",
-                            Email = "manager@pethome.app",
-                            EmailConfirmed = true,
-                            LockoutEnabled = true,
-                            NormalizedEmail = "MANAGER@PETHOME.APP",
-                            NormalizedUserName = "MANAGER@PETHOME.APP",
-                            PasswordHash = "AQAAAAEAACcQAAAAEIVDPe+D2k9DYm8aaZlygE1ycvP2BRWCG5m/08oxAWV/yXRq84CDZIl+GZQBVp7rTA==",
-                            PhoneNumberConfirmed = false,
-                            SecurityStamp = "60032225-177c-4d2c-bfad-f31b1988dad2",
-                            TwoFactorEnabled = false,
-                            UserName = "manager@pethome.app"
-                        });
```

Comment:
this is part of the last migration, it's the behavior that we are moving to the seeddata.  That is why I reverted it

Reply:
Right, it was a revert. I wasn't sure that version number was being provided by a library.